configs/analysis_configs/feature_select_configs.yaml

```diff
@@ -6,18 +6,25 @@ feature_select_configs:
     operation:
       - variance_threshold
       - drop_na_columns
-      - correlation_threshold
-      # - blocklist This requires a blocklist file. (meeting with greg)
+      # - correlation_threshold
```

Comment:
typo?

Reply:
I commented it out in order for the cell health dataset to work. An error kept riseing when using correlation_threshold 

I'll put that in the issues.